libraries/codegen-tooling/inc/can_msg_defs.h

```diff
@@ -62,6 +62,7 @@ typedef enum {
   SYSTEM_CAN_MESSAGE_CRUISE_CONTROL_COMMAND = 41,
   SYSTEM_CAN_MESSAGE_AUX_DCDC_VC = 43,
   SYSTEM_CAN_MESSAGE_DCDC_TEMPS = 44,
+  SYSTEM_CAN_MESSAGE_UV_CUTOFF_NOTIFICATION = 45,
```

Comment:
Also make a PR in codegen-tooling-msxiv, I'll merge it at the same time as this gets merged.